### PR TITLE
Rename $ => $262. Fixes gh-802

### PR DESCRIPTION
**Pull request by rwaldron**
Signed-off-by: Rick Waldron <waldron.rick@gmail.com>

### CODE REVIEW

**Comment by rwaldron** [comment]
@bterlson I think you'll need to make changes here: https://github.com/bterlson/eshost

**Comment by bterlson** [comment]
@rwaldron plan is to add a config key for the identifier name and test262-harness will set it to `$262`. I want to still use `$` for most usages inside eshost.

**Comment by rwaldron** [comment]
Sounds good to me

**Comment by jugglinmike** [comment]
Looks good to me!

**Comment by bakkot** [comment]
Ping @littledan; Chromium's testing harness will need to be changed too.

**Comment by littledan** [comment]
LGTM

**Comment by leobalter** [comment]
This is good to go for now. 

I also used the github's Resolve Conflicts feature for the first time in here, it's hopefully ok.

File: INTERPRETING.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ properties of the global scope prior to test execution.
 - **`print`** A function that exposes the string value of its first argument to
   the test runner. This is used as a communication mechanism for asynchronous
   tests (via the `async` flag, described below).
-- **`$`** An ordinary object with the following properties:
+- **`$262`** An ordinary object with the following properties:
   - **`createRealm`** - a function which creates a new [ECMAScript
     Realm](https://tc39.github.io/ecma262/2016/#sec-code-realms),
     defines this API on the new realm's global object, and returns the `$`
```
Comment (edgemaster) [inline INTERPRETING.md]: I think that a change to `$262` was missed here.

Comment (lars-t-hansen) [inline INTERPRETING.md]: I agree.

**Pull request merged**